kernel/src/thread/status.rs

```diff
@@ -27,10 +27,6 @@ impl ThreadStatus {
     pub fn is_exited(&self) -> bool {
         *self == ThreadStatus::Exited
     }
-
-    pub fn is_stopped(&self) -> bool {
-        *self == ThreadStatus::Stopped
-    }
```

Comment:
So should we remove `ThreadStatus`? It seems that `ThreadStatus::Init` may not be very useful and `ThreadStatus::Stopped` is no longer useful after this PR.